server/liveblog/blogs/blogs.py

```diff
@@ -194,7 +203,10 @@ def on_updated(self, updates, original):
         recipients = []
         for user in members:
```

Comment:
I remember I tried to put it into a single method but for some reasons it was keep failing and I let it aside. I could try again right now.